res/database/Default/database_news.xml

```diff
@@ -8617,17 +8335,19 @@ W tym roku ponownie spodziewane są demonstracje.</pl>
 				<de>Nach historischem Vorbild wollen ${swedenandnorway} eine Union bilden. Auch ${finland} würde sich der neuen USL (Union Skandinavischer Länder) anschließen.</de>
 				<en>Following historical precedent, ${swedenandnorway} want to form a union. ${finland} also would join the new USN (Union of Scandinavian Nations).</en>
 			</description>
-			<variables> <!-- Select one of five Scandinavian countries, then select 2 of the remaining 4; maybe there's a simpler way to do this. -->
-				<i>0|1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29</i>
-				<j>${.csv:"0;0;0;1;1;1;2;2;2;3;3;3;4;4;4;5;5;5;6;6;6;7;7;7;8;8;8;9;9;9":i}</j>
-				<finland>
-					<de>${.csv:"Finnland;Dänemark;Island;Norwegen;Finnland;Island;Norwegen;Dänemark;Island;Schweden;Finnland;Island;Schweden;Dänemark;Island;Schweden;Norwegen;Island;Dänemark;Norwegen;Finnland;Norwegen;Schweden;Finnland;Schweden;Finnland;Dänemark;Norwegen;Schweden;Dänemark":i}</de>
-					<en>${.csv:"Finland;Denmark;Iceland;Norway;Finland;Iceland;Norway;Denmark;Iceland;Sweden;Finland;Iceland;Sweden;Denmark;Iceland;Sweden;Norway;Iceland;Denmark;Norway;Finland;Norway;Sweden;Finland;Sweden;Finland;Denmark;Norway;Sweden;Denmark":i}</en>
-				</finland>
+			<variables>
+				<countries_csv>
+					<de>Schweden;Norwegen;Finnland;Dänemark;Island</de>
+					<en>Sweden;Norway;Finland;Denmark;Iceland</en>
+				</countries_csv>
+				<combination_csv>0;1;2|0;1;3|0;1;4|0;2;3|0;2;4|0;3;4|1;2;3|1;2;4|1;3;4|2;3;4</combination_csv>
```

Comment:
Shouldn't there be more combinations (I don't mean all)? 0 seems to be overrepresented as first country.

Reply:
Maybe this is something a new function "could" resolve ... something like a `${.numberset:start:end:elementcount[:setcount:setdelimiter]}`?

Or is this something so rarely used that there is no need for it?

Reply:
I think, it is not a very common pattern. And what if you don't really want all possible combinations?
A simple .randomInt:start:end is higher on the list.

Reply:
I think it's correct. Same variation count as before. Each should be used 6 times. It's just that the ordering is always the same. What *could* be done in addition is variable ordering too.

Reply:
From the origninal version, I thought the order is relevant - select one contry, then select two of the remaining. So at least the first country should have the same number of occurences for all 5.